bears/jinja2/Jinja2Bear.py

```diff
@@ -112,11 +112,16 @@ class Jinja2Bear(LocalBear):
 
     VARIABLE_REGEX = re.compile(
         r'(?P<open>{{)(?P<content>.*?)(?P<close>}})')
+    STATEMENT_REGEX = re.compile(
+        r'(?P<open>{%[+-]?)'
+        # Ignore PycodestyleBear (E501) LineLengthBear
```

Comment:
nah nah, please break the string properly ;)

Reply:
@Makman2 Please provide some more detail on how exactly should i improve this. I am not able to get it clearly.

Reply:
-->
```
(r'string-that-is-ver'
 r'y-long')
```
:)